### PR TITLE
Remove the t3 lockout

### PR DESCRIPTION
**Pull request by Stop-Signs**
## About the PR
Removed the lockout on T3 research. also forces you to get all techs in any given tier before moving on.

## Why / Balance
Epi always getting t3 industrial just so they can have funny bags and powercells. this creates an issue where adding t3 tech (or anything service or arsenal related) is useless as epi will almost never research it.
## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:

- remove: Removed the T3 Lockout on epi techs

### CODE REVIEW

**Comment by Avalon-Proto** [comment]
Now we just need to make it so they need to research x amount of T1/T2 to level up tech

**Comment by Stop-Signs** [comment]
> Now we just need to make it so they need to research x amount of T1/T2 to level up tech

This is already a thing, i could just raise the value

**Comment by Stop-Signs** [comment]
Oh this also fixes an upstream error where you cant put research disks in the server without your dev env crashing

**Review by MilonPL — CHANGES_REQUESTED** [review]
noob change the lockout tier to 4 inside of the prototypes instead of whatever this is

**Comment by MilonPL** [comment]
> Oh this also fixes an upstream error where you cant put research disks in the server without your dev env crashing

pr it upstream then

**Comment by Stop-Signs** [comment]
> > Oh this also fixes an upstream error where you cant put research disks in the server without your dev env crashing
> 
> pr it upstream then

its already pr'd upstream ;)

**Comment by MilonPL** [comment]
cherry pick in a separate PR

File: Content.Shared/Research/Prototypes/TechDisciplinePrototype.cs
Comment (MilonPL) [inline Content.Shared/Research/Prototypes/TechDisciplinePrototype.cs:1]: set this in yaml

File: Resources/Prototypes/Research/disciplines.yml
Comment (MilonPL) [inline Resources/Prototypes/Research/disciplines.yml:1]: deltav comments

**Review by MilonPL — CHANGES_REQUESTED** [review]

**Comment by SolStar2** [comment]
> cherry pick in a separate PR

[This is said pr](https://github.com/space-wizards/space-station-14/pull/33205) Its already marked as approved so I imagine it won't be long before it's here anyways.

File: Resources/Prototypes/Research/disciplines.yml
Comment (MilonPL) [inline Resources/Prototypes/Research/disciplines.yml:1]: deltav comments

File: Resources/Prototypes/Research/disciplines.yml [hunk 1]
```diff
@@ -5,10 +5,11 @@
   icon:
     sprite: Interface/Misc/research_disciplines.rsi
     state: industrial
+  lockoutTier: 4
```
Comment (MilonPL) [inline Resources/Prototypes/Research/disciplines.yml]: .

File: Resources/Prototypes/Research/disciplines.yml [hunk 2]
```diff
@@ -17,10 +18,11 @@
   icon:
     sprite: Interface/Misc/research_disciplines.rsi
     state: arsenal
+  lockoutTier: 4
```
Comment (MilonPL) [inline Resources/Prototypes/Research/disciplines.yml]: .

File: Resources/Prototypes/Research/disciplines.yml [hunk 3]
```diff
@@ -29,10 +31,11 @@
   icon:
     sprite: Interface/Misc/research_disciplines.rsi
     state: experimental
+  lockoutTier: 4
```
Comment (MilonPL) [inline Resources/Prototypes/Research/disciplines.yml]: .

File: Resources/Prototypes/Research/disciplines.yml [hunk 4]
```diff
@@ -41,7 +44,8 @@
   icon:
     sprite: Interface/Misc/research_disciplines.rsi
     state: civilianservices
+  lockoutTier: 4
```
Comment (MilonPL) [inline Resources/Prototypes/Research/disciplines.yml]: .

File: Content.Shared/Research/Prototypes/TechDisciplinePrototype.cs
Comment (MilonPL) [inline Content.Shared/Research/Prototypes/TechDisciplinePrototype.cs:1]: this file doesn't need any changes

**Review by MilonPL — DISMISSED** [review]

**Review by Colin-Tel — DISMISSED** [review]
Looks good to me and the curators I've asked.

File: Content.Shared/Research/Prototypes/TechDisciplinePrototype.cs [hunk 5]
```diff
@@ -1,4 +1,4 @@
-﻿using Robust.Shared.Prototypes;
+using Robust.Shared.Prototypes;
```
Comment (deltanedas) [inline Content.Shared/Research/Prototypes/TechDisciplinePrototype.cs]: untouch

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by Spielern** [comment]
Epi, with the recent RE changes, has become a lot more stale. I just had a round 30 minutes ago where I had Svalinns, Microreactor Cells, Speed Boots, and Bluespace bags off of a single research server. The only thing stopping people from doing this every shift is the occasional RA that goes unchecked and raises glimmer by 500 with artifacts. This just seems insanely overpowered. I already dislike how epi's balance is, and I can see the department getting way more unbalanced and unfun with this. This could be a unique opinion to me, but I play an insane amount of epi and have heard similar discontent from other epi players in game through LOOC.

**Review by deltanedas — APPROVED** [review]

**Review by Colin-Tel — APPROVED** [review]

**Comment by Stop-Signs** [comment]
> Epi, with the recent RE changes, has become a lot more stale. I just had a round 30 minutes ago where I had Svalinns, Microreactor Cells, Speed Boots, and Bluespace bags off of a single research server. The only thing stopping people from doing this every shift is the occasional RA that goes unchecked and raises glimmer by 500 with artifacts. This just seems insanely overpowered. I already dislike how epi's balance is, and I can see the department getting way more unbalanced and unfun with this. This could be a unique opinion to me, but I play an insane amount of epi and have heard similar discontent from other epi players in game through LOOC.

Not once in here did you mention the other t3's that get ignored by epi. bluespace syringes and beakers, as well as the advanced laser pistol and the Advanced truncheon are all things that epi doesn't benefit greatly from, but other departments do. the problem this PR seeks to solve is epistemics researching only for themselves instead of for everyone. If cloning got added to the civillian services tree as a t3, do you think epi would get that instead of their designer bags and needlessly infinite power cells?

**Comment by Spielern** [comment]
> Not once in here did you mention the other t3's that get ignored by epi. bluespace syringes and beakers, as well as the advanced laser pistol and the Advanced truncheon are all things that epi doesn't benefit greatly from, but other departments do. the problem this PR seeks to solve is epistemics researching only for themselves instead of for everyone. If cloning got added to the civillian services tree as a t3, do you think epi would get that instead of their designer bags and needlessly infinite power cells?

I didn't mention any of the other T3s because I believe we should be heading the opposite way of this PR with Epi. IMO, T3 technologies were designed to be extremely good and rare, not something the crew is entitled to.
A large reason that I have seen Industrial being the only T3 chosen recently is BECAUSE of the fact that speed boots are able to be obtained for free. Before that, it was really just a coin flip depending on who is playing. I'd see arsenal and civilian both be picked fairly often, although Industrial still was the most common. Before the arsenal changes, all of the T3s except civilian benefit Epi AND the crew. People would just choose whatever they felt like in the moment, or whatever RA #3 accidentally pressed. I see no issue with how it was before, and still completely regret the RE changes. 
If your sole goal is to make Epi less selfish with their T3 picks, I firmly believe reverting the RE additions would be much more productive for both sides.

**Comment by deltanedas** [comment]
- bluespace beakers have basically no use for epi, and limited use for med since chems arent infinite and if you are farming oxygen you will quickly run into the sink generation speed limit
- so many jobs benefit from speedboots, for one sec
- realistically only salv needs bags of holding, everyone else can manage a satchel within reason, bar atmos or paramed storing their suits
- engi, med, sec, salv all benefit from microreactors which is why theyre such a strong pick already
- t3 arsenal is solely for warops and syndies emagging lathes, you never see it otherwise
- t3 experimental is a 100% meme that benefits noone. half the time its made is just for exploiting 1 of 1000 bugs with the qsi, theres very little utility for anyone

**Pull request merged**